### PR TITLE
basic pong with only one mode and no elo

### PR DESCRIPTION
**Pull request by tsiguenz**

### CODE REVIEW

File: frontend/src/views/Game.vue [hunk 1]
```diff
@@ -1,14 +1,166 @@
...
+      this.ctx.fill();
+    },
+    writeScore() {
+      this.ctx.font = '20px Poppins';
+      this.ctx.fillStyle = 'white';
+      this.ctx.fillText(this.score.player1.points, this.map.width / 2 - 25, 20);
+      this.ctx.fillText(this.score.player2.points, this.map.width / 2 + 15, 20);
+    },
+    handleKeyDown(e) {
+      if (e.keyCode === 38) {
```
Comment (Crysicia) [inline frontend/src/views/Game.vue:128]: Please name the keycode instead of using a magic number :3

Comment (tsiguenz) [reply frontend/src/views/Game.vue:128]: Magic is cool

**Review by Crysicia — APPROVED** [review]
Seems good, everything work as expected

**Review by gmorange42 — APPROVED** [review]
Looks good, Well Done !

**Pull request merged**